src/common/lang/en.js

```diff
@@ -114,10 +114,10 @@ module.exports = {
   addressDetail: {
     name: 'ADDRESS DETAILS',
     address: 'Address: ',
-    ontBalance: 'ONT Balance: ',
-    ongBalance: 'ONG Balance: ',
-    claimable: 'Claimable ONG: ',
-    unbound: 'Unbound ONG: ',
+    ontBalance: 'ONYX Balance: ',
+    ongBalance: 'OXG Balance: ',
+    claimable: 'Claimable ONYX: ',
```

Comment:
ONT -> ONYX and ONG -> OXG ? If so, there is mistake

Reply:
fixed